### DISCUSSION TITLE
QGIS Crashes when exporting MBTiles

### DISCUSSION THREAD

**Discussion by alpinfra** [opengisch/QField#2289] [Q&A]
QGIS version 3.22.0 crashes when exporting a project to QField using the QFieldSync Tool

QGIS version 3.16.4 finishes the job successfully.

**Comment by m-kuhn** [comment]
Hi @alpinfra thank you for reporting this.
Could you open a new issue at https://github.com/opengisch/qfieldsync/issues and leave relevant information there (operating system, error messages, configuration, data).
Thanks in advance
